getting-started/jdbc/docker-compose.yml

```diff
@@ -29,18 +29,22 @@ services:
       # Optional, allows attaching a debugger to the Polaris JVM
       - "5005:5005"
     environment:
-      - JAVA_DEBUG=true
-      - JAVA_DEBUG_PORT=*:5005
-      - POLARIS_PERSISTENCE_TYPE=relational-jdbc
-      - POLARIS_PERSISTENCE_RELATIONAL_JDBC_MAX_RETRIES=5
-      - POLARIS_PERSISTENCE_RELATIONAL_JDBC_INITIAL_DELAY_IN_MS=100
-      - POLARIS_PERSISTENCE_RELATIONAL_JDBC_MAX_DELAY_IN_MS=5000
-      - QUARKUS_DATASOURCE_DB_KIND=pgsql
-      - QUARKUS_DATASOURCE_JDBC_URL=${QUARKUS_DATASOURCE_JDBC_URL}
-      - QUARKUS_DATASOURCE_USERNAME=${QUARKUS_DATASOURCE_USERNAME}
-      - QUARKUS_DATASOURCE_PASSWORD=${QUARKUS_DATASOURCE_PASSWORD}
-      - POLARIS_REALM_CONTEXT_REALMS=POLARIS
-      - QUARKUS_OTEL_SDK_DISABLED=true
+      JAVA_DEBUG: true
+      JAVA_DEBUG_PORT: "*:5005"
+      POLARIS_PERSISTENCE_TYPE: relational-jdbc
+      POLARIS_PERSISTENCE_RELATIONAL_JDBC_MAX_RETRIES: 5
+      POLARIS_PERSISTENCE_RELATIONAL_JDBC_INITIAL_DELAY_IN_MS: 100
+      POLARIS_PERSISTENCE_RELATIONAL_JDBC_MAX_DELAY_IN_MS: 5000
+      QUARKUS_DATASOURCE_DB_KIND: pgsql
+      QUARKUS_DATASOURCE_JDBC_URL: $QUARKUS_DATASOURCE_JDBC_URL
+      QUARKUS_DATASOURCE_USERNAME: $QUARKUS_DATASOURCE_USERNAME
+      QUARKUS_DATASOURCE_PASSWORD: $QUARKUS_DATASOURCE_PASSWORD
+      POLARIS_REALM_CONTEXT_REALMS: POLARIS
+      QUARKUS_OTEL_SDK_DISABLED: true
+      POLARIS_BOOTSTRAP_CREDENTIALS: POLARIS,${CLIENT_ID},${CLIENT_SECRET}
+      polaris.features."ALLOW_INSECURE_STORAGE_TYPES": "true"
+      polaris.features."SUPPORTED_CATALOG_STORAGE_TYPES": "[\"FILE\",\"S3\",\"GCS\",\"AZURE\"]"
```

Comment:
For another pr: Maybe this should only be FILE and we have a built in error message for users trying to use any other storage type to come back and redo their config. Not a huge deal. 

Also we probably should not allow "ALLOW_INSECURE_STORAGE_TYPES" if an insecure storage type is not in supported storage types right? Also "ignore-severe-issues" probably shouldn't be allowed to be true unless a sever item has been selected.

Just thoughts for future cleanups though

Reply:
Ack, I allowlisted all as this same script also used with cloud providers,.

> Also we probably should not allow "ALLOW_INSECURE_STORAGE_TYPES" if an insecure storage type is not in supported storage types right? Also "ignore-severe-issues" probably shouldn't be allowed to be true unless a sever item has been selected.

Sure thing ! let me take a deeper look in this as fast followup